src/styles/card.scss

```diff
@@ -0,0 +1,84 @@
+.card {
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  background: $white;
+  border: 1px solid $gray-elements;
```

Comment:
The variable `$gray-elements` is used here. Ensure that it is defined in your SCSS files or imported from another file.